### PR TITLE
feat(project): Enable publication on maven

### PR DESCRIPTION
**Pull request by jhart1685**
Signed-off-by: James Hart <jhart@uk.ibm.com>

## PR summary
This PR enables publication of artifacts to Maven central

**Fixes:** <! -- link to issue -->

## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [x] Build/CI related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## What is the current behavior?  
SDK behaviour is unchanged

## What is the new behavior?  
SDK behaviour is unchanged

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
             of the SDK generator used to generate your SDK code.
             See this link for details: https://github.ibm.com/CloudEngineering/openapi-sdkgen/wiki/Compatibility-Chart
         -->
-        <sdk-core-version>9.5.4</sdk-core-version>
+        <sdk-core-version>9.9.0</sdk-core-version>
```
Comment (padamstx) [inline pom.xml:29]: We're up to 9.9.1 now :)

**Review by padamstx — APPROVED** [review]
The changes look good as far as I can tell.

File: pom.xml (hunk 1 above)
Comment (jhart1685) [reply pom.xml:29]: Yep, thanks - 9.9.0 matches the generator version used here though.

**Review by bainsy88 — APPROVED** [review]
LGTM

**Pull request merged**